docs/overview.qmd

```diff
@@ -3,232 +3,560 @@ title: Overview
+    ui.input_select("var2", None, choices=["tip", "total_bill"], width="100%")
+## file: requirements.txt
+palmerpenguins
+plotly
+pandas
 ```
+
 :::
 
-Notice how Shiny uses nested function calls to indicate parent/child relationships.
-In this example, `ui.input_radio_buttons()` is inside of `ui.panel_sidebar()`, and `ui.panel_sidebar()` is in `ui.layout_sidebar()`, and so on.
+Shiny also knows when outputs are visible or not, and so, will only call `render` functions when needed.
+For example, in the app below, the `table` function doesn't get called until the "Table" page is selected.
+
+::: column-body-outset-right
+
+```{shinylive-python}
+#| standalone: true
+#| components: [editor, viewer]
+#| layout: vertical
+#| viewerHeight: 325
+import plotly.express as px
+from shiny.express import input, render, ui
+from shinywidgets import render_plotly
+
+tips = px.data.tips()
 
-This example also includes some explanatory text written in a [Markdown](https://en.wikipedia.org/wiki/Markdown) string literal, and uses the [`ui.markdown()`](../api/reference/shiny.ui.markdown.html) function to render it to HTML.
+with ui.sidebar():
+    ui.input_selectize("var", "Select variable", choices=["total_bill", "tip"])
 
-## Shiny UI is HTML
+ui.nav_spacer()
 
-It's worth noting at this point that Shiny UI is entirely made up of HTML.
-Each of the nested function calls in the previous section returns a snippet of HTML.
+with ui.nav_panel("Plot"):
+    @render_plotly
+    def plot():
+        p = px.histogram(tips, x=input.var())
+        p.update_layout(height=225)
+        return p
+
+with ui.nav_panel("Table"):
+    @render.data_frame
+    def table():
+        return tips[[input.var()]]
+## file: requirements.txt
+palmerpenguins
+plotly
+pandas
+```
 
-It's not important that you know HTML to make good use of Shiny, but if you do know HTML, this fact may be helpful in forming a mental model for how Shiny UI works.
+:::
+
+::: callout-note
+#### Learn more
+
+For a more in-depth look at reactivity, check out the [reactivity article](reactive-foundations.qmd).
+:::
 
-``` {shinylive-python}
-#| components: [editor, cell]
-from shiny import ui
+### Starter templates {#templates}
 
-ui.input_numeric("n", "N", 0)
+Once you've [installed](install-create-run.qmd) Shiny, the `shiny create` CLI command provides access to a collection of useful starter templates. This command walks you through a series of prompts to help you get started quickly with a helpful example. One great option is the dashboard template, which can be created with:
+
+```bash
+shiny create -t dashboard
 ```
 
-Adding a `ui.panel_sidebar` call around `ui.input_numeric` simply wraps the control with additional HTML (a `<div>` and `<form>`, in this case).
-The nesting of the generated HTML matches the nesting of the Python function calls in your UI.
+![The resulting dashboard generated by the dashboard template](assets/dashboard-template.png){class="img-shadow"}
```

Comment:
I don't think this screenshot matches the template so we should change the photo.